config/config.go

```diff
@@ -345,7 +376,7 @@ func parseServerIssuerURL(sType ServerType) error {
 		return errors.New("If Server.IssuerHostname is configured, you must provide a valid port")
 	}
 
-	if sType == OriginType {
+	if IsServerEnabled(OriginType) {
```

Comment:
What occurs if multiple servers are running from the same process? If I start up an origin and then a cache -- will this still show true when I start the cache? That's a problem since we don't want the cache to execute this code block.

I suggest going back to having a "this is the server calling the function" variable and a separate "enabled server" variable, the latter of which can be used by the web uis.